Source/WebInspectorUI/UserInterface/Views/ScriptClusterTimelineView.js

```diff
@@ -53,7 +53,12 @@ WI.ScriptClusterTimelineView = class ScriptClusterTimelineView extends WI.Cluste
         this._selectedTarget = null;
         this._displayedTarget = (!this.representedObject.imported && targets.includes(WI.mainTarget)) ? WI.mainTarget : (targets.firstValue || WI.assumingMainTarget());
 
-        this._pathComponentForTarget = new Map(targets.map((target) => [target, this._createTargetPathComponent(target)]));
+        this._pathComponentForTarget = new Map;
+        for (let target of targets) {
+            let pathComponent = this._createTargetPathComponent(target);
+            pathComponent.selectorArrows = target.size > 1;
```

Comment:
Should this be `targets.size`? Or is it `pathComponents.size`?

Reply:
oops yup